dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/AlertDao.java

```diff
@@ -99,31 +97,31 @@ public int updateAlert(AlertStatus alertStatus, String log, int id) {
     /**
      * query user list by alert group id
      *
-     * @param alerGroupId alerGroupId
+     * @param alertGroupId alertGroupId
      * @return user list
      */
-    public List<User> queryUserByAlertGroupId(int alerGroupId) {
+    public List<User> queryUserByAlertGroupId(int alertGroupId) {
```

Comment:
I think alert group and user no longer have any relationship in this feature. So this method need be delete.